### PR TITLE
Add sequenced entries to entry bundles in MySQL storage implementation

### PR DESCRIPTION
**Pull request by roger2hk**
#21

### CODE REVIEW

File: storage/mysql/mysql.go [hunk 1]
```diff
@@ -291,6 +300,56 @@ func (s *Storage) integrate(ctx context.Context, tx *sql.Tx, fromSeq uint64, ent
...
+		}
+	}
+
+	// Add new entries to the bundle.
+	for _, e := range sequencedEntries {
+		if _, err := bundleWriter.Write(e.BundleData); err != nil {
+			return fmt.Errorf("bundleWriter.Write: %w", err)
+		}
+		entriesInBundle++
+		fromSeq++
```
Comment (AlCutter) [inline storage/mysql/mysql.go:330]: If you increment `fromSeq` here, will your call to `tb.Integrate` below start building/updating the Merkle tree at the wrong size?

**Review by mhutchinson — APPROVED** [review]
It would be great to see some in-memory SQL tests for this soon.

File: storage/mysql/mysql.go [hunk 2]
```diff
@@ -188,6 +190,15 @@ func (s *Storage) ReadEntryBundle(ctx context.Context, index uint64) ([]byte, er
 	return entryBundle, row.Scan(&entryBundle)
 }
 
+func (s *Storage) writeEntryBundle(ctx context.Context, tx *sql.Tx, index uint64, entryBundle []byte) error {
+	if _, err := tx.ExecContext(ctx, replaceTiledLeavesSQL, index, entryBundle); err != nil {
+		klog.Errorf("Failed to execute replaceTiledLeavesSQL: %v", err)
```
[the remark this answers is not in the document]
Comment (roger2hk) [reply storage/mysql/mysql.go:195]: This is the full displayed message which should be capitalized.

```
On the other hand, the style for the full displayed message (logging, test failure, API response, or other UI) depends, but should typically be capitalized.
```

https://github.com/google/styleguide/blob/gh-pages/go/decisions.md#error-strings

**Comment by roger2hk** [comment]
> It would be great to see some in-memory SQL tests for this soon.

Sure, I will add the tests this week.

**Pull request merged**